packages/rancher-pushprox/charts/templates/pushprox-clients.yaml

```diff
@@ -26,8 +26,11 @@ spec:
       {{- end }}
       containers:
       - name: pushprox-client
-        image: arvindiyengar/pushprox-linux-amd64:add_flag_for_token_path
-        command: [ "/app/pushprox-client" ]
+        image: {{ .Values.clients.image }}
```

Comment:
While we're making changes, we should add in support for a private registry. Also, I've seen some charts separate out between image and tag in their templating. Should we do the same?

Reply:
I agree with separating out the image tag vs. the templating; will update the PR accordingly for that.

Also, there are a set of minor changes that also need to be addressed like private registry, updating resource limits, etc. so I'm going to change this PR into one that addresses several minor fixes for the `rancher-monitoring` package.

Will be reflected on next push

Reply:
Pending in latest push: private registry changes